packages/network/samba/scripts/samba-autoshare

```diff
@@ -21,6 +21,8 @@ if [ -f /storage/.cache/services/samba.conf ]; then
   . /storage/.cache/services/samba.conf
 
   if [ "$SAMBA_AUTOSHARE" == "true" ] ; then
-    systemctl restart smbd.service
```

Comment:
Good suggestion, that also gives the user a clean way of reloading after manual changes to the configuration. I'll test it and revise.